orchagent/intfsorch.cpp

```diff
@@ -289,6 +365,42 @@ void IntfsOrch::doTask(Consumer &consumer)
             {
                 vnet_name = value;
             }
+            else if (field == "mtu")
+            {
+                try
+                {
+                    mtu = static_cast<uint32_t>(stoul(value));
+                }
+                catch (const std::invalid_argument &e)
+                {
+                    SWSS_LOG_ERROR("Invalid argument %s to %s()", value.c_str(), e.what());
+                    continue;
+                }
+                catch (const std::out_of_range &e)
+                {
+                    SWSS_LOG_ERROR("Out of range argument %s to %s()", value.c_str(), e.what());
+                    continue;
+                }
+
+                // Inherit mtu from parent port or port channel
+                mtu = 0;
```

Comment:
Isn't this the same issue as I mentioned above? Overwrite?

Reply:
yes. When mtu == 0, the mtu is inherited from the parent port/port channel.

Reply:
Removed